App.js

```diff
@@ -1,7 +1,19 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import { StyleSheet, Text, View } from 'react-native';
 
 export default function App() {
+  const URL = 'https://pro-api.coinmarketcap.com/v1/cryptocurrency/listings/latest';
```

Comment:
there is no need attaching it to a react component, just make it a function that can be called by any component at any time